### PR TITLE
#2443 - Metrics - Gen-Mapper Multiplier Version Of The Dynamic Metrics

### PR DESCRIPTION
**Pull request by kodinkat**
- fixes: #2443 
---

Screenshot 2024-05-20 at 16 08 15

### CODE REVIEW

**Comment by kodinkat** [comment]
Screenshot 2024-05-21 at 14 14 01

**Comment by corsacca** [comment]
@kodinkat no longer available in the dynamic metrics section?
image

File: dt-metrics/metrics.php [hunk 1]
```diff
@@ -36,11 +36,10 @@ public function __construct(){
...
                 $modules = dt_get_option( 'dt_post_type_modules' );
 
-                if ( dt_has_permissions( [ 'multiplier' ] ) ) {
-                    /* Record Types */
-                    $genmap_base_slug = 'personal';
-                    $genmap_base_title = __( 'Personal', 'disciple_tools' );
-                    require_once( get_template_directory() . '/dt-metrics/records/genmap.php' );
+                require_once( get_template_directory() . '/dt-metrics/records/genmap.php' );
+
+                if ( dt_has_permissions( [ 'multiplier' ] ) || dt_has_permissions( [ 'dt_all_access_contacts', 'view_project_metrics' ] ) ) {
```
Comment (corsacca) [inline dt-metrics/metrics.php:41]: @kodinkat there is no 'multiplier' permission.
multiplier is a role.
I don't think a permission check here for "personal" metrics is needed.

**Comment by corsacca** [comment]
@kodinkat clicking on Personal > generation map opens the menu to the dynamic section: 
image

**Pull request merged**